internal/pkg/template/templates/addons/ddb/env/access_policy.yml

```diff
@@ -0,0 +1,54 @@
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+  Name:
+    Type: String
+    Description: Your workload's name.
+Resources:
+  {{logicalIDSafe .Name}}AccessPolicy:
+    Metadata:
+      'aws:copilot:description': 'An IAM ManagedPolicy for your service to access the {{.Name}} db'
+    Type: AWS::IAM::ManagedPolicy
+    Properties:
+      Description: !Sub
+        - Grants CRUD access to the Dynamo DB table ${Table}
+        - Table: { Fn::ImportValue: { Fn::Sub: "${App}-${Env}-{{logicalIDSafe .Name}}TableName" } }
+      PolicyDocument:
+        Version: '2012-10-17'
+        Statement:
+          - Sid: DDBActions
+            Effect: Allow
+            Action:
+              - dynamodb:BatchGet*
+              - dynamodb:DescribeStream
+              - dynamodb:DescribeTable
+              - dynamodb:Get*
+              - dynamodb:Query
+              - dynamodb:Scan
+              - dynamodb:BatchWrite*
+              - dynamodb:Create*
+              - dynamodb:Delete*
+              - dynamodb:Update*
+              - dynamodb:PutItem
+            Resource: 
+              Fn::ImportValue: !Sub "${App}-${Env}-{{logicalIDSafe .Name}}ARN"
+          - Sid: DDBLSIActions
+            Action:
+              - dynamodb:Query
+              - dynamodb:Scan
+            Effect: Allow
+            Resource: !Sub
+              - ${ TableARN }/index/*
+              - TableARN: { Fn::ImportValue: { Fn::Sub: "${App}-${Env}-{{logicalIDSafe .Name}}ARN" }}
+
+Outputs:
+  {{envVarName .Name}}:
```

Comment:
Since all addon templates get merged into one mega template, I wonder if this output name is unique enough?

Reply:
Done! I suffixed the ddb outputs' logical ID with `"DdbTableName"` and `"DdbTableAccessPolicy"`, similarly with s3 bucket's `Outputs`.

Also added a comment about what is the env var name they will use to access the value from the service main container.